cumulusci/tasks/apex/testrunner.py

```diff
@@ -493,6 +491,16 @@ def _enqueue_test_run(self, class_ids):
             method="POST", url=self.tooling.base_url + "runTestsAsynchronous", json=body
         ).json()
 
+    def _init_task(self):
```

Comment:
Clarification: this positioning is so that the value does not get frozen for MetaDeploy, correct?